experiments/operational_domain/operational_domain_3d_bestagon.cpp

```diff
@@ -0,0 +1,208 @@
+//
+// Created by marcel on 08.08.23.
```

Comment:
@marcelwa Please double-check the dates in the experiment files.

Reply:
Thanks, fixed!